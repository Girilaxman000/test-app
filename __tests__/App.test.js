

import React from 'react';
import renderer from 'react-test-renderer';
import App from "../App";

describe('test an app', () => {
    test('render an app correctly', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})
