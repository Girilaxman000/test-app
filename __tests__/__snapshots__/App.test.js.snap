// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test an app render an app correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hi welcome to react native app.
  </Text>
</View>
`;
